diplomacy/server/notifier.py

```diff
@@ -362,6 +362,11 @@ def notify_game_message(self, server_game, game_message):
         """
         if game_message.is_global():
             yield self._notify_game(server_game, notifications.GameMessageReceived, message=game_message)
+            for game_role, token in server_game.get_omniscient_addresses():
+                yield self._notify(notifications.GameMessageReceived(token=token,
+                                                                     game_id=server_game.game_id,
+                                                                     game_role=game_role,
+                                                                     message=game_message))
```

Comment:
What does this change do?

Reply:
This allows omniscient addresses who has connection established (i.e., admin) to receive real-time message updates.